src/PowerShell/Help/Microsoft.WinGet.Client/Disable-WinGetSetting.md

```diff
@@ -0,0 +1,94 @@
+﻿---
+external help file: Microsoft.WinGet.Client.Cmdlets.dll-Help.xml
+Module Name: Microsoft.WinGet.Client
+online version:
+schema: 2.0.0
+---
+
+# Disable-WinGetSetting
+
+## SYNOPSIS
+Disables an administrative setting.
+
+## SYNTAX
+
+```
+Disable-WinGetSetting [-Name] <String> [-ProgressAction <ActionPreference>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Disables an administrative setting. Requires administrator access.
+
+Administrator Settings default to "Disabled" and may be controlled by Group Policy Objects.
+
+Administrator Settings:
+
+* LocalManifestFiles
+* BypassCertificatePinningForMicrosoftStore
+* InstallerHashOverride
+* LocalArchiveMalwareScanOverride
+* ProxyCommandLineOptions
+* DefaultProxy
```

Comment:
Looking at the source code, `DefaultProxy` appears to be a [string admin setting](https://github.com/microsoft/winget-cli/blob/6df6ded70ab3b558a54f764ba8ed47968350b37f/src/AppInstallerCommonCore/Public/winget/AdminSettings.h#L11-L28) and does not work with `Disable-WinGetSetting` or `winget settings --disable DefaultProxy`

```
~ winget settings --disable DefaultProxy
Windows Package Manager v1.8.1911
Copyright (c) Microsoft Corporation. All rights reserved.

The value provided for the `disable` argument is invalid; valid values are: LocalManifestFiles, BypassCertificatePinningForMicrosoftStore, InstallerHashOverride, LocalArchiveMalwareScanOverride, ProxyCommandLineOptions
```